test/lit.cfg

```diff
@@ -311,6 +311,40 @@ if test_sdk_overlay_dir is not None:
 test_options = os.environ.get('SWIFT_TEST_OPTIONS')
 config.swift_test_options = ' ' + test_options if test_options else ''
 
+def get_clang_library_dir():
+    llvm_lib_clang = os.path.join(config.llvm_libs_dir, 'clang')
+    if not os.path.exists(llvm_lib_clang):
+        return ''
+
+    versions = os.listdir(llvm_lib_clang)
+    if not len(versions):
+        return ''
+
+    version = max(versions)
+    return os.path.join(llvm_lib_clang, version, 'lib',
+                        platform.system().lower())
+
+config.clang_library_dir = get_clang_library_dir()
```

Comment:
No, lib/swift/clang/ should just contain libclang* if compiler-rt is present. I guess that means tweaking the `swift_install_in_components` command that handles the headers today.

IIRC there's also a build mode where it just makes a symlink, which would be good enough for testing.